NotificationHubs/src/com/windowsazure/messaging/NativeNotification.java

```diff
@@ -0,0 +1,52 @@
+package com.windowsazure.messaging;
+
+import org.apache.http.entity.ContentType;
+
+import java.util.Map;
+
+/**
+ * This interface represents a native notification.
+ */
+public interface NativeNotification {
```

Comment:
So reading over [these docs](https://docs.microsoft.com/en-us/rest/api/notificationhubs/send-apns-native-notification), it looks like we use "native notification" to distinguish from a "template notification".  I don't mind the terminology; but I had to do a little digging to understand where it came from. Mind updating the interface comment to say something along the lines of "This interface represents a notification to send to a specific platform such as APNS, FCM, or WNS. It is distinguished from a template notification."

Reply:
Removed interface to avoid confusion